EIPS/eip-3290.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 3290
+title: Time Based Switch
+author: Branislav Djalic (@Omodaka9375), Malisa Pusonja (@horohronos), Ivor Jugo (@ivorrr)
+discussions-to: https://github.com/ethereum/EIPs/issues/3286
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2021-02-23
+---
+
+## Simple Summary
+This EIP provides standard interfaces and processes for safe, conditional transactions based on user predefined timestamps. It would enable users to secure their accounts from lockout after account activity ceased for a predefining fallback address to transfer the funds.
```

Comment:
Simple summary should be a similar length to an email subject line, pull request title line, etc.
```suggestion
Processes for safe, conditional transactions based on user predefined timestamps.
```